orders/src/main/resources/templates/orders.html

```diff
@@ -4,36 +4,69 @@
 <head>
     <meta charset="UTF-8">
     <title>Order</title>
+    <link rel="stylesheet" href="css/orders.css">
 </head>
-<body>
-<H1><span th:text="'Welcome '+${username}+' to the ordering page'"></span></H1>
-<H2>please place your order here</H2>
-
-
-<form action="#" th:action="@{/order}" th:object="${orders}" method="post">
-    <table id="orders_table">
+<body style="font-family:'David',sans-serif">
+<div style="text-align: center;">
+    <H1><span th:text="${username}+' ברוך הבא '"></span></H1>
+    <H2>אנא הוסף את המוצרים הרצויים להזמנה</H2>
+    <table class="tg" id="orderTable">
         <thead>
         <tr>
-            <th>מוצר</th>
-            <th>ברקוד</th>
-            <th>מחיר ליחידה</th>
-            <th>כמות</th>
+            <th class="tg-cscf">מחלקה</th>
+            <th class="tg-cscf">קוד מוצר</th>
+            <th class="tg-cscf">שם מוצר</th>
+            <th class="tg-cscf">מחיר ליחידה/100 גרם</th>
+            <th class="tg-cscf">כמות להזמנה</th>
+            <th class="tg-cscf">סה"כ מחיר</th>
         </tr>
         </thead>
         <tbody>
-        <tr th:each="order, itemStat : *{orders}">
-            <input type="hidden" th:field="*{orders[__${itemStat.index}__].name}">
-            <input type="hidden" th:field="*{orders[__${itemStat.index}__].barcode}">
-            <input type="hidden" th:field="*{orders[__${itemStat.index}__].price}">
-            <td th:text="*{orders[__${itemStat.index}__].name}"></td>
-            <td th:text="*{orders[__${itemStat.index}__].barcode}"></td>
-            <td th:text="*{orders[__${itemStat.index}__].price}"></td>
-            <td><input th:field="*{orders[__${itemStat.index}__].quantity}"></td>
+        <tr>
+            <td class="tg-0pky">
+                <label for="selectDepartment"></label><select id="selectDepartment" onchange="departmentSelected()">
+                <option>בחר מחלקה...</option>
+            </select>
+            </td>
+            <td class="tg-0pky"><label for="enterBarcode"></label><input type="text" id="enterBarcode"
+                                                                         onchange="getProductFromBarcode()"></td>
+            <td class="tg-0pky"><label for="chooseProductName"></label><select id="chooseProductName"
+                                                                               onchange="getProductFromName()"></select>
+            </td>
+            <td class="tg-0pky"></td>
+            <td class="tg-0pky"><label for="enterAmount"></label><input type="number" id="enterAmount"
+                                                                        onchange="amountChosen()"></td>
+            <td class="tg-0pky"></td>
         </tr>
         </tbody>
     </table>
-    <input type="submit"/>
-</form>
-
+    <br>
+    <br>
+    <button id="addProductBtn" onclick="addProduct()">הוסף להזמנה</button>
+    <br>
+    <br>
+    <br>
+    <H2>Order Summary</H2>
```

Comment:
we need a test for this table.
Add one to the static HTML page tests, and we should sometimes find a way to unit test the JS itself (but that's a bit later, probably)

Reply:
Not sure how to write tests for HTML... i'll leek at what you have and try.

Reply:
For the time being, just add something to the clientSideTest - there's already something testing the other table, you can add something very similar

Reply:
right now there is just the OrdersControllerClientSideTest, and it runs tests on a table named "orderForm". There is no "orderForm" in the HTML - there are "orderTable" and "orderSummaryTable". Do you want me to change the tests to the correct name and duplicate them for the second table? I'll try and add some tests for the different selects and buttons.

Reply:
The test should fail if there are no such elements (they are failing in the CI as you can see), so yes - please fix them

Test fails can be seen here: 
https://travis-ci.org/amitwer/shelachOrders/builds/574477676?utm_source=github_status&utm_medium=notification


relevant message: 
[ERROR] Failures: 
[ERROR]   OrdersControllerClientSideTest.loginReturnsProductTable:80->verifyOrdersTable:89 
Expecting actual not to be null
[ERROR]   OrdersControllerClientSideTest.loginReturnsProductTable:80->verifyOrdersTable:89 
Expecting actual not to be null
[INFO] 
[ERROR] Tests run: 12, Failures: 2, Errors: 0, Skipped: 0